src/app/server/Server.h

```diff
@@ -416,6 +417,8 @@ class Server
 
     app::reporting::ReportScheduler * GetReportScheduler() { return mReportScheduler; }
 
```

Comment:
Newly introduced code from Server.h/BUILD.gn should be guarded by CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC (introduced by https://github.com/project-chip/connectedhomeip/pull/38749)

Reply:
I've created an issue to track this: [#38988](https://github.com/project-chip/connectedhomeip/issues/38988)